Core/GDCore/IDE/Events/EventsRefactorer.cpp

```diff
@@ -711,6 +728,24 @@ bool EventsRefactorer::SearchStringInConditions(
   return false;
 }
 
+bool EventsRefactorer::SearchStringInComments(gd::ObjectsContainer& project,
+                                               gd::ObjectsContainer& layout,
+                                               gd::EventsList& events,

```

Comment:
Pass a `gd::BaseEvent& event`. You don't need to pass a list of event, just the event where to search in.